pkg/actors/actors_test.go

```diff
@@ -904,18 +734,6 @@ func TestCreateReminder(t *testing.T) {
 	assert.Equal(t, numReminders, len(reminderReferences))
 	assert.Equal(t, numReminders, len(reminders))
 
-	// For the rest of the test, we're disabling bulk get in the fake state store
-	store, ok := testActorsRuntime.compStore.GetStateStore("actorStore")
```

Comment:
Why is this removed?

Reply:
All state stores now implement `BulkGet`. This test was asserting for state stores that returned `false` from `BulkGet`, so it's unnecessary.